std/math.d

```diff
@@ -328,6 +335,9 @@ template floatTraits(T)
             enum EXPPOS_SHORT = 0;
             enum SIGNPOS_BYTE = 0;
         }
+
+        import std.typecons : Tuple;
+        alias Integral = Tuple!(ulong, "significand", ushort);
```

Comment:
Is this alignment issues because of ``Tuple(ushort, ulong)`` or because of the accessor?  I would be very surprised if ``Tuple`` doesn't pack data, that would defeat the purpose of us using ``Tuple`` here in the first place.